sentry_sdk/integrations/beam.py

```diff
@@ -133,38 +134,36 @@ def _inner(*args, **kwargs):
     return _inner  # type: ignore
 
 
-def _capture_exception(exc_info, hub):
-    # type: (ExcInfo, Hub) -> None
+def _capture_exception(exc_info):
+    # type: (ExcInfo) -> None
     """
     Send Beam exception to Sentry.
     """
-    integration = hub.get_integration(BeamIntegration)
-    if integration is None:
-        return
+    client = sentry_sdk.get_client()
 
-    client = hub.client
-    if client is None:
+    integration = client.get_integration(BeamIntegration)
+    if integration is None:
         return
 
     event, hint = event_from_exception(
         exc_info,
         client_options=client.options,
         mechanism={"type": "beam", "handled": False},
     )
-    hub.capture_event(event, hint=hint)
+    sentry_sdk.capture_event(event, hint=hint)
 
 
 def raise_exception(client):
     # type: (Optional[BaseClient]) -> None
     """
-    Raise an exception. If the client is not in the hub, rebind it.
+    Raise an exception. If the client is not set, rebind it.
     """
-    hub = Hub.current
-    if hub.client is None:
-        hub.bind_client(client)
+    if not sentry_sdk.get_client().is_active():
+        Scope.get_global_scope().set_client(client)
```

Comment:
this is not necessary because `sentry_sdk.get_client()` above always returns a client (if not active it is a `NonRecordingClient`). So I guess we can remove the client from this function completely.

Reply:
Removed